CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.0.10

```

Comment:
I think is a good idea to move to the increase the minor (major.minor.revision) every time we update the Devtools version (to a new major)

Reply:
Sounds good.  I don't think we have been doing that before, but we might as well start now.